cmd/tfimport/main.go

```diff
@@ -29,6 +29,20 @@ import (
 	"github.com/GoogleCloudPlatform/healthcare-data-protection-suite/internal/tfimport"
 )
 
+// From https://stackoverflow.com/a/28323276
+type mapFlag map[string]bool
+
+func (i *mapFlag) String() string {
+	return "my string representation"
```

Comment:
Is this supposed to be a repr of the map?

Maybe use json.Marshal?

Reply:
Yes, I forgot to change this line. Will fix it.